samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -29,13 +29,8 @@ bucket = '<bucket-name>'
 
 # Load the dataset
 s3 = boto3.client("s3")
-data_bucket = S3DataConfig(
-    sm_session, "example-notebooks-data-config", "config/data_config.json"
-).get_data_bucket()
-print(f"Using data from {data_bucket}")
-
 s3.download_file(
-    data_bucket, "datasets/image/MNIST/mnist.pkl.gz", "mnist.pkl.gz")
+    "sagemaker-sample-files", "datasets/image/MNIST/mnist.pkl.gz", "mnist.pkl.gz")
```

Comment:
Great! Can you please check if it can be changed here as well? so all places use the same bucket
https://github.com/kubeflow/pipelines/blob/master/samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py#L90

Only restriction with the above is it has a region restriction. All instructions are written assuming it can be run in `us-east-1`